tadrep/visuals.py

```diff
@@ -19,6 +23,17 @@
 CONTIG_HEAD_MODIFIER = 200                                                  # Modifier of contig head length
 
 
+def create_plots():
+    json_files = cfg.output_path.glob('*.json')
```

Comment:
Why multiple `JSON` files? All information of a project (therefore all plasmid related information) can be stored within a single file.